lib/core/src/Cardano/Wallet/Primitive/AddressDerivation/Random.hs

```diff
@@ -135,3 +152,36 @@ hashSeed = serialize . blake2b256 . serialize
 
 blake2b256 :: ScrubbedBytes -> ScrubbedBytes
 blake2b256 = BA.convert . hash @ScrubbedBytes @Blake2b_256
+
+-- | Compute passphrase as hash of the root public key.
+deriveHDPassphrase :: Key 'RootK XPub -> Passphrase "random"
```

Comment:
this function is almost the same as in Rodneys' #587

Reply:
I'd avoid `derive` as a name here since this is rather associated with key derivation already. `toHdPassphrase` or `hdPassphrase` is probably fine.

Reply:
now I work on top of #587